.env.sample

```diff
@@ -1,14 +1,15 @@
 OPENCTI_ADMIN_EMAIL=admin@opencti.io
-OPENCTI_ADMIN_PASSWORD=ChangeMe_STRING
-OPENCTI_ADMIN_TOKEN=ChangeMe_UUIDv4
-OPENCTI_BASE_URL=http://opencti.ChangeMe.com_STRING
-MINIO_ROOT_USER=ChangeMeAccess_STRING
-MINIO_ROOT_PASSWORD=ChangeMeKey_STRING
-RABBITMQ_DEFAULT_USER=guest
-RABBITMQ_DEFAULT_PASS=guest
-CONNECTOR_EXPORT_FILE_STIX_ID=ChangeMe1_UUIDv4
-CONNECTOR_EXPORT_FILE_CSV_ID=ChangeMe2_UUIDv4
-CONNECTOR_EXPORT_FILE_TXT_ID=ChangeMe3_UUIDv4
-CONNECTOR_IMPORT_FILE_STIX_ID=ChangeMe4_UUIDv4
-CONNECTOR_IMPORT_DOCUMENT_ID=ChangeMe5_UUIDv4
-SMTP_HOSTNAME=ChangeMe.Mail.Com
+OPENCTI_ADMIN_PASSWORD=changeme
+OPENCTI_ADMIN_TOKEN=6f42a3da-5c97-48e7-8ee4-ba3d0164daf0
```

Comment:
I don't think it's a good idea to fix the token here.

Reply:
I reverted the token and add a note to `README.md` in order to remember new users to set a UUID token.